Units/parser-qemuhx.r/enable.d/args.ctags

```diff
@@ -1,4 +1,4 @@
 --sort=no
 --extras-QemuHX=+{funcmap}
 --fields=+E
-
+--language-force=QemuHX
```

Comment:
Though this should be fixed, I would like you to delete this modification from this commit?
Can I ask you?

Reply:
Hmm, not really, the test fails without this

Reply:
Oh, I see why this change is needed.
Ignore what I wrote.

Reply:
I have to provide a command-line interface to define an arbitrator resolving this kind of conflict in the future